mlir/include/Catalyst/Transforms/Passes.td

```diff
@@ -93,6 +93,14 @@ def QnodeToAsyncLoweringPass : Pass<"qnode-to-async-lowering"> {
     let constructor = "catalyst::QnodeToAsyncLoweringPass()";
 }
 
+def DisableAssertionPass : Pass<"disable-assertion"> {
+    let summary = "Disable all catalyst_assertions.";
+
+    let dependentDialects = ["LLVM::LLVMDialect"];
```

Comment:
This is rather minor, but dependant dialect declarations for passes are only required for dialects created by the pass (which aren't already in the input IR). Since this pass does not generate any ops, no dependant dialects need to be declared :)
```suggestion
```

Reply:
Updated in new commit